README.md

```diff
@@ -1,23 +1,71 @@
-# Devito: Fast Finite Difference Computation from Symbolic Specification
+# Devito: Fast Stencil Computation from Symbolic Specification
 
 [![Build Status for the Core backend](https://github.com/devitocodes/devito/workflows/CI-core/badge.svg)](https://github.com/devitocodes/devito/actions?query=workflow%3ACI-core)
 [![Build Status with MPI](https://github.com/devitocodes/devito/workflows/CI-mpi/badge.svg)](https://github.com/devitocodes/devito/actions?query=workflow%3ACI-mpi)
 [![Code Coverage](https://codecov.io/gh/devitocodes/devito/branch/master/graph/badge.svg)](https://codecov.io/gh/devitocodes/devito)
+[![Slack Status](https://img.shields.io/badge/chat-on%20slack-%2336C5F0)](https://opesci-slackin.now.sh)
+
+[Devito](http://www.devitoproject.org) is a Python package to implement
```

Comment:
>Devito is a Python package to implement optimized stencil computation from high-level symbolic problem definitions.

Some people (e.g. certain applied mathematicians) are not going to understand exactly what you mean by 'stencil computation'. They'd understand finite difference though - don't you think we should mention FD in some form in this opening paragraph also?

Reply:
yes. The reason I switched to stencil was for the exact opposite reason -- to avoid scaring away the non applied mathematicians.

So "stencil" is the superclass. How about adding:

_Devito is a Python package to implement optimized stencil computation (e.g., finite differences, image processing, machine learning) from high-level symbolic problem definitions._

? or something along the lines. Ideas welcome, and yes, I agree with you we should work in this direction

Reply:
Yes, the above sentence looks good to me.

Reply:
done